docs/docs/recipes.md

```diff
@@ -192,45 +187,34 @@ since `0.7.4`).  Requires Mesos version `0.20.0` or later.
-That is desirable because it truly makes your infrastructure easily scalable. All resources are treated equally. 
-However, there are times when you want
-to separate some machines from the common pool and ascertain that only some special tasks are scheduled there. For
-example,
+Usually, Marathon decides where to start your application tasks without your input. This allows your infrastructure to scale easily, since all resources are treated equally.
+
+However, there are times when you want to separate some machines from the common pool and schedule only certain tasks on them. For example:
+
+* Some nodes in your cluster have publicly reachable network addresses and you only want to start tasks there that make use of public addresses.
 
-* you might have some nodes with publicly reachable network addresses. You only want to start tasks there
-  that make use of public addresses.
-* you have some nodes with special hardware such as very fast SSDs or GPUs that you do not want to waste for
-  regular tasks.
+* Some nodes in your cluster have special hardware, such as very fast SSDs or GPUs, that you want to reserve for tasks that need this hardware.
   
 ### Using roles
 
-You can configure some of your Mesos agents such that they offer their resources only to a specific
-Mesos role. Only Mesos frameworks (such as Marathon) that are configured for this specific Mesos role will get
-offers for these resources. The advantage is that you can prevent accidental usage of these resources 
-by other Mesos frameworks this way.
+You can configure some of your Mesos agents to only offer their resources to a specific Mesos role. Only Mesos frameworks (such as Marathon) that are configured for this specific Mesos role will get offers for these resources. In this way, you can prevent accidental usage of these resources by other Mesos frameworks.
 
-Configuring Marathon: `--mesos_role yourrole`
+1. Configure Marathon: `--mesos_role yourrole`
 
-Configuring Mesos Master: `--roles=<...other roles...>,yourrole`
+2. Configure the Mesos master: `--roles=<...other roles...>,yourrole`
 
-Configuring the Mesos Agents in question: Either use `--default_role yourrole` to assign all resources of that
-agent to your role or use the `--resources` configuration to assign individual resources to that role (such
-as a certain port range). See
-[the Mesos command line documentation](http://mesos.apache.org/documentation/latest/configuration/) for details.
+3. Configuring the Mesos agents whose resource offers you want to restrict. Either use the  `--default_role yourrole` flag to assign all that agent's resources to your role or use the `--resources` flag to assign individual resources to that role (such as a certain port range). See [the Mesos command line documentation](http://mesos.apache.org/documentation/latest/configuration/) for details.
 
-### Preventing Accidental Usage of Special Roles inside Marathon
+### Preventing Accidental Use of Special Roles inside Marathon
```

Comment:
See below what the author meant by accidental use.  I think the wording can be improved.